core/tx_pool.go

```diff
@@ -209,6 +209,7 @@ func (pool *TxPool) resetState() {
 	pool.promoteExecutables(currentState)
 }
 
+// Stop stops a TxPool
```

Comment:
Please add a period to the end of the comment, and write out `transaction pool`, not `TxPool`.